Documentation/DocuBlocks/Rest/Administration/get_admin_log.md

```diff
@@ -214,172 +214,169 @@ API is enabled, accessing it requires admin privileges, or even superuser
 
 @RESTBODYPARAM{agency,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{agencycomm,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{agencystore,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{aql,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{arangosearch,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{audit-authentication,string,optional,string}
-One of the possible log levels (_Enterprise Edition only_).
+One of the possible log topics (_Enterprise Edition only_).
 
 @RESTBODYPARAM{audit-authorization,string,optional,string}
-One of the possible log levels (_Enterprise Edition only_).
+One of the possible log topics (_Enterprise Edition only_).
 
 @RESTBODYPARAM{audit-collection,string,optional,string}
-One of the possible log levels (_Enterprise Edition only_).
+One of the possible log topics (_Enterprise Edition only_).
 
 @RESTBODYPARAM{audit-database,string,optional,string}
-One of the possible log levels (_Enterprise Edition only_).
+One of the possible log topics (_Enterprise Edition only_).
 
 @RESTBODYPARAM{audit-document,string,optional,string}
-One of the possible log levels (_Enterprise Edition only_).
+One of the possible log topics (_Enterprise Edition only_).
 
 @RESTBODYPARAM{audit-hotbackup,string,optional,string}
-One of the possible log levels (_Enterprise Edition only_).
+One of the possible log topics (_Enterprise Edition only_).
 
 @RESTBODYPARAM{audit-service,string,optional,string}
-One of the possible log levels (_Enterprise Edition only_).
+One of the possible log topics (_Enterprise Edition only_).
 
 @RESTBODYPARAM{audit-view,string,optional,string}
-One of the possible log levels (_Enterprise Edition only_).
+One of the possible log topics (_Enterprise Edition only_).
 
 @RESTBODYPARAM{authentication,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{authorization,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{backup,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{cache,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{cluster,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{clustercomm,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{collector,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{communication,string,optional,string}
-One of the possible log levels.
```

Comment:
should this not be replaced?

Reply:
You mean log level > log topic? It is, see the added (green) line 277. It's a little obfuscated by the removal of the compactor topic.